newdle/client/src/util/timezones.js

```diff
@@ -0,0 +1,43 @@
+import preval from 'preval.macro';
+import 'moment-timezone';
+
+// This will just generate a pre-compiled list with all timezones and their
+// respective offsets.
+export const commonTimezones = preval`
+  require('moment-timezone');
+  const moment = require('moment');
+  const _ = require('lodash');
+
+  const tzData = require('moment-timezone/data/meta/latest.json');
+  const commonTzSet = Object.entries(tzData.countries).reduce((accum, [key, {zones}]) => {
+    zones.forEach(tz => {
+      accum[tz] = true;
+    })
+    return accum;
+  }, {});
+
+  // add some common timezones which are not official in any country
+  [
+    // UTC/GMT
+    'UTC',
```

Comment:
I don't see UTC in the dropdown (nor any of the other timezones from this list)

Reply:
Weird. I can see them.

Reply:
Maybe it's because I force-pushed them a few minutes after the PR was opened?

Reply:
OK maybe the babel cache is to blame... maybe it didn't rebuild the file after I checked out your updated branch... after having been on a different branch for a while and switching back to yours it worked